CHANGELOG.md

```diff
@@ -17,6 +19,54 @@
 
 # OpenFunction
 
+## 0.7.0-rc.0 / 2022-08-11
+The ofn CLI install method was deprecated.
+
+The `domains.core.openfunction.io` CRD was deprecated and removed.
+
+### Features
+
+- [Add the parameter validation capabilities for Function](https://github.com/OpenFunction/OpenFunction/issues/274). #290
+- Add OpenFunction Gateway [proposal](https://github.com/OpenFunction/OpenFunction/blob/main/docs/proposals/202207-openfunction-gateway.md). #314
```

Comment:
Gateway proposal could be one of the PR in `Add Gateway & Route for OpenFunction`